src/client/client.spec.ts

```diff
@@ -77,9 +77,9 @@ describe("Client:", () => {
       expect(client._events.hasOwnProperty('testevent2')).toBe(true);
     });
 
-    it("should only be able to add strings as new possible event", () => {
+    it('should only be able to add strings as new possible event', () => {
       let numOfKeysBefore = Object.keys(client._events).length;
-      let eventTypes = [1, [], {}, 1.2, undefined, null, true];
+      let eventTypes = [1, [], {}, 1.2, undefined, undefined, true];
```

Comment:
this null was here as a test case for one of the different types of data that could be thrown at it